### DISCUSSION TITLE
Sharing of Scenario Trends graph

### DISCUSSION THREAD

**Discussion by johnqa** [ludeknovy/jtl-reporter#240] [Ideas]
Hi, 

It's great to be able to share reports but it would be wonderful if we could also share Scenario Trends graph for a quick overview in time of how our AUT performs.

Thanks,
John

**Comment by ludeknovy** [comment]
Hi @johnqa !

How would you like to be able to share it - should this be done automatically or manually? Using the same notification channels as there are for the report sharing? Or just generating it manually and downloading it?

**Reply by johnqa** [reply]
Hi,

I think the same mechanism as for report sharing would be ok. 
I don't want to download it, but to have it displayed as an iframe in a dashboard.

Thanks

**Reply by ludeknovy** [reply]
Hi, I was thinking about how this could work, I could think of two options: 
1) Introduce a new share token for the scenario level. If a token is provided in the url param, it would show the scenario page, where only the scenario chart would be shown, all actions such as uploading a new report would be disabled.
2) Build external iframe, that would load the data using the api token and display it in a chart.

What would work better for you?

**Reply by johnqa** [reply]
Hi,

Thank you for the effort.

I think option number 2 would work best for me.

Regards,
John

**Reply by ludeknovy** [reply]
Ok, the final solution is going to be a mix of the two:
<iframe src="http://localhost:4200/project/test-project/scenario/blabla/items?token=<token>" width="1000" height="800"></iframe>


![Screenshot 2023-10-25 at 11 28 52](https://github.com/ludeknovy/jtl-reporter/assets/13610612/a1f2a81b-d9bc-4a8c-97b4-8cf1c65fa1f8)

**Reply by johnqa** [reply]
that looks great, thanks

**Reply by ludeknovy** [reply]
https://github.com/ludeknovy/jtl-reporter/assets/13610612/810007bd-cacb-47fc-a9a9-690d567e9013

**Reply by ewallz** [reply]
How to share the scenario along with its test runs results?

**Reply by ludeknovy** [reply]
Hi @ewallz there's currently no way to do this.
